### DISCUSSION TITLE
Redis ttl support (SET with EX)?

### DISCUSSION THREAD

**Discussion by maslowivan** [caching-tools/next-shared-cache#454] [Ideas]
Easiliy to add the TTL (SET with EX) option to prevent the memory overflow. Should we?))

**Comment by better-salmon** [comment]
@studentIvan hello! Thank you for your question! Let me add some context.

To be more consistent `@neshca/cache-handler` operates with absolute time. Redis does support `EXAT` option for `SET` command since version 6.2.0. And there is no such option for `RedisJSON` at all.

Maybe it is a good idea to use `EXAT` instead of calling `EXPIRE` command for `redis-string` Handler. But we will drop any Redis setups prior to 6.2.0.

Do you have suggestions about `RedisJSON` e.g. `redis-stack` Handler?

**Reply by maslowivan** [reply]
Redis stack FT.SEARCH command does not supported in aws elasticache, so we have to use the redis-strings on highload projects. @better-salmon hi. Thanks for answer.

**Comment by better-salmon** [comment]
@studentIvan take a look on this PR #473.
